ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/MappedOperationPool.java

```diff
@@ -89,6 +100,52 @@ public MappedOperationPool(
             OPERATION_POOL_SIZE_VALIDATION_REASON + metricType,
             "Total number of attempts to add an operation to the pool, broken down by validation result",
             "result");
+
+    this.timeProvider = timeProvider;
+    asyncRunner.runWithFixedDelay(
+        this::updateLocalSubmissions,
+        Duration.ofMinutes(10),
+        this::updateLocalSubmissionsErrorHandler);
+  }
+
+  private void updateLocalSubmissionsErrorHandler(Throwable throwable) {}
```

Comment:
Not sure if the intent was to do nothing with the error in this handler. If that is the case I would suggest renaming it to make it clear that the intent is to do nothing (or maybe adding a `//DO NOTHING` comment in the body).

Reply:
should really log it, may leave at debug initially